java/src/main/native/src/ChunkedReaderJni.cpp

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <memory>
+#include <vector>
+
+#include <cudf/column/column.hpp>
+#include <cudf/io/parquet.hpp>
+#include <cudf/table/table.hpp>
+
+#include "../include/jni_utils.hpp"
```

Comment:
Appear only `#include "jni_utils.hpp"` to be enough.

Reply:
Include with double quote and no relative path should be applied only to the header file located in the same folder as the current file.

Reply:
Hmm. Why not `#include <jni_utils.hpp>`?

This is moot, actually. `<cudf_jni_apis.hpp>` includes `<jni_utils.hpp>`.

Reply:
Yeah this works :)

Reply:
Humnn, after trying I think I should do what @firestarman suggested to consistent with what did in other files. Otherwise, the clang-format will mess up the headers order. The clang-format style sheet for our JNI C++ is really bad.

Reply:
AFAIK, include with double quote will search header file first under the same path with src, if not find, will search under system include pathes.